src/socket/tcp.rs

```diff
@@ -152,7 +156,7 @@ const RTTE_MAX_RTO: u32 = 10000;
 
 #[derive(Debug, Clone, Copy)]
 #[cfg_attr(feature = "defmt", derive(defmt::Format))]
-struct RttEstimator {
+pub struct RttEstimator {
```

Comment:
why does this have to be pub?

Reply:
Because this implementation let users to pass a congestion controller via a type parameter,
`RttEstimator` must be exposed.
However, I will fix to remove the type parameter as discussed above,
and it will be non-public.

Reply:
fixed